src/commands/test/lib/nunit-xml-util.ts

```diff
@@ -11,29 +11,45 @@ export class NUnitXmlUtil extends XmlUtil {
     const tempPath: string = await pfs.mkTempDir("appcenter-uitestreports");
     let mainXml: Document = null;
 
-    const self = this;
+    const self: NUnitXmlUtil = this;
     return new Promise<Document>((resolve, reject) => {
       fs.createReadStream(pathToArchive)
         .pipe(unzip.Parse())
         .on("entry", function (entry: unzip.Entry) {
-          const fullPath = path.join(tempPath, entry.path);
+          // Skip directories and hidden system files
+          if (entry.type === "Directory" || path.basename(entry.path).substring(0, 1) === ".") {
```

Comment:
Is there anything we can do to refactor the shared code here and in `junit-xml-util`?

Reply:
Will do this